cypress/integration/upload-a-document.spec.ts

```diff
@@ -18,6 +18,8 @@ describe('Can upload a document', () => {
   });
 
   it('lets you choose a file', () => {
+    cy.visit(`http://localhost:3000/resident/foo`);
+    cy.get('a').contains('Continue').click();
```

Comment:
How come these changes are in this PR, they’re already in main. I think it needs a rebase/merge with main?

Reply:
i did [one yesterday](https://github.com/LBHackney-IT/document-evidence-store-frontend/pull/13/commits/0156b2c485c8a8de4ca21f5085838a9f2d1ea41a) but they went away again somehow